tlatools/org.lamport.tlatools/src/tla2sany/semantic/OpDefNode.java

```diff
@@ -1196,6 +1197,32 @@ public final boolean getOpLevelCond(int i, int j, int k) {
     return this.opLevelCond[i][j][k];
   }
 
+  /**
+   * Returns an array giving the maximum levels of the parameters to this
+   * operator, or null if no such information is available.
+   * 
+   * @return The maximum arg levels.
+   */
+  public final int[] getArgMaxLevels() {
+    return
+      this.maxLevels == null
+      ? null
+      : Arrays.copyOf(this.maxLevels, this.maxLevels.length);
```

Comment:
Is returning a copy done primarily as a precaution? If so, let's add a comment explaining this to help our future selfs understand it.